README.md

```diff
@@ -51,14 +51,16 @@ FATA[0000] Post http:///var/run/docker.sock/v1.17/images/create?fromImage=cockro
 ```
 On OSX:
 ```bash
-# Setup Boot2Docker. This should only need to be done once.
-boot2docker init
-# Start Boot2Docker. This will need to be run once per reboot.
-boot2docker start
-# Setup environment variables. This will need to be run once per shell.
-$(boot2docker shellinit)
+# After installing [Docker Toolbox](https://www.docker.com/toolbox), 
```

Comment:
these instructions are already in the context of the person running their own shell and running into the ssl issue. I think all we need is to:
- make sure the vm is running: `docker-machine start default`
- initialize the shell: `eval $(docker-machine env default)`